fluentd/vendored_gem_src/syslog_protocol/lib/syslog_protocol/packet.rb

```diff
@@ -23,6 +40,15 @@ def assemble(max_size = 1024)
       data
     end
 
+    def assemble_rfc5424()
```

Comment:
no need to supply parens here if you are not passing args

Reply:
updated